src/core/lang.c

```diff
@@ -295,6 +311,10 @@ const uint8_t *lang_get_string(int group, int index)
     if (group == 96 && !index) {
         return translation_for(TR_BUILDING_SMALL_TEMPLE_VENUS_NAME);
     }
+    //TODO: Move to proper group and translation
+    if (group == 97 && !index) {
```

Comment:
I have no idea how these magical groups are working and what should be here :-) Happy to hear any suggestion.

Reply:
Translation groups are indices are indeed magic. No one ever went ahead and created an enum for them.

We find what we want by trial and error, basically!